examples/skr/aci-arm-template.json

```diff
@@ -30,7 +30,16 @@
               "command": [
                 "/skr.sh"
               ],
-              "image": "mcr.microsoft.com/aci/skr:2.2",

```

Comment:
This does not look right. I can't see what image this container is supposed to use now.